presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,164 @@
+* Only request HTTPS client certificate when certificate authentication is enabled. ({issue}`4804`)
+* Add user mapping configuration to preserve lower case or upper case user names. ({issue}`4736`)
+
+## Web UI
+
+* Fix displaying physical input read time in detailed query view. ({issue}`4962`)
+
+## JDBC Driver Changes
+
+* Implement `ResultSet.getStatement()`. ({issue}`4957`)
+
+## BigQuery Connector Changes
+
+* Add support for HOURLY partitioning of tables. ({issue}`4968`)
+* Redact value of `bigquery.credentials-key` from logs. ({issue}`4968`)
+
+## Cassandra Connector Changes
+
+* Map Cassandra `TIMESTAMP` type to Presto `TIMESTAMP(3) WITH TIME ZONE` type. ({issue}`2269`)
+
+## Hive Connector Changes
+
+* Skip stripes and row group based on timestamp statistics for ORC files. ({issue}`1147`)
+* Skip DeepArchive objects when `skip-glacier-objects` is set. ({issue}`5002`)
+* Use temporary staging directory for write operation to sorted bucketed table. ({issue}`3434`)
+* Invalidate cached information on `GRANT` and `REVOKE`. ({issue}`4768`)
+* Add Parquet and RCBinary [configuration properties](hive_configuration_properties) `hive.parquet.time-zone` and `hive.rcfile.time-zone` to adjust binary timestamp
+  values to a specific time zone. For Hive 3.1+, this should be set to UTC. The default value is the JVM default time zone, for
+  backwards compatibility with earlier versions of Hive. ({issue}`4799`)
+* Add ORC [configuration property](hive_configuration_properties) `hive.orc.time-zone` to set the default time zone for legacy ORC files that did not declare a time
+  zone. ({issue}`4799`)
+* Replace the `hive.time-zone` configuration property with the format specific properties. ({issue}`4799`)
+* Allow to optionally use cluster role in s3 mapping. ({issue}`4931`)
+* Remove support for bucketing on timestamp. ({issue}`4759`)
+* Improve performance of partition fetching from Glue. ({issue}`4938`)
+* Fix invalid timestamp values for nested data in Text, Avro, Sequence File, JSON and CSV formats. ({issue}`4799`)
+* Fix query failure when reading a ORC ACID table with a filter, after the table underwent a minor table compaction. ({issue}`4622`)
+* Fix issue where queries could fail if Rubix caching was used and cached data was evicted during query execution. ({issue}`3580`)
+* Fix incorrect results when filtering on columns of `TIMESTAMP` type. ({issue}`5083`)
+
+## Iceberg Connector Changes
+
+* Skip stripes and row group based on Timestamp statistics for ORC files. ({issue}`1147`)
+
+## JMX Connector Changes
+
+* Change `timestamp` column type in history tables to `TIMESTAMP WITH TIME ZONE`. ({issue}`4753`)
+
+## Kafka Connector Changes
+
+* Preserve time zone when parsing `TIMESTAMP WITH TIME ZONE` values. ({issue}`4799`)
+
+## Kinesis Connector Changes
+
+* Preserve time zone when parsing `TIMESTAMP WITH TIME ZONE` values. ({issue}`4799`)
```

Comment:
Here & elsehwere -- I think we have always written Presto types as ``` `lowercase` ```, haven't we?

Reply:
Not really. We've been inconsistent about it.